### PR TITLE
[Program: GCI] Add pending requests' count endpoint

### LINKED ISSUE #320: Create an API Endpoint to Count Pending Mentorship Relations
**Describe the solution you'd like**
Create a new API Endpoint at GET/mentorship_relations/count_pending which counts the number of pending mentorship relations of the current user. Return a dictionary response with key "count" and value as the number of pending mentorship relations.

**Additional context**
This issue has been created for students participating in GCI'19.

### PR DESCRIPTION
**Pull request by ilyamaier**
### Description
- Added new count pending requests model
- Added new DAO method and a new endpoint `/mentorship_relations/count_pending`

Fixes #320

### Type of Change:
- Code

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
The code was tested on my laptop.

### Screenshots:

`/mentorship_relations/count_pending:`

Screenshot (50)

`/mentorship_relations/count_pending (without a call, only description):`

Screenshot (51)

`/mentorship_relations/pending (first part): `

Screenshot (52)

`/mentorship_relations/pending (second part): `

Screenshot #4

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Update Postman API at /docs folder
- [x] Update Swagger documentation and the exported file at /docs folder

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/api/dao/mentorship_relation.py [hunk 1]
```diff
@@ -365,3 +365,29 @@ def list_pending_mentorship_relations(user_id):
...
+            success if pending mentorship relation requests' count is returned,
+            failure if otherwise.
+        """
+
+        user = UserModel.find_by_id(user_id)
+        now_timestamp = datetime.now().timestamp()
+        pending_requests = 0
+        all_relations = user.mentor_relations + user.mentee_relations
+
+        for relation in all_relations:
```
Comment (annabauza) [inline app/api/dao/mentorship_relation.py]: have you ever tried lamda expressions with python?

Comment (ilyamaier) [reply app/api/dao/mentorship_relation.py]: not really. I will Google it now. Or maybe you could explain that to me?

Comment (ilyamaier) [reply app/api/dao/mentorship_relation.py]: @annabauza could you say me, what exactly do you want to do with a lambda in this function?

Comment (ilyamaier) [reply app/api/dao/mentorship_relation.py]: I know what a lambda is, but I don't know exactly how to use it here.

Comment (ilyamaier) [reply app/api/dao/mentorship_relation.py]: I guess, I understand what you want to do. Could you give me a minute?

Comment (ilyamaier) [reply app/api/dao/mentorship_relation.py]: @annabauza, I used lambda here.

**Review by annabauza — CHANGES_REQUESTED** [review]
Hey,
I think swagger.json should not be modify manually.

**Comment by ilyamaier** [comment]
> Hey,
> I think swagger.json should not be modify manually.

@annabauza yes. Sorry! My mistake. I've changed that.

**Review by annabauza — APPROVED** [review]

**Comment by isabelcosta** [comment]
Thank you for your contribution!
I will close this PR, because this is a not a desirable feature for the app.

**Pull request closed without merging**